README.md

```diff
@@ -1,69 +1,101 @@
-<a href="https://colab.research.google.com/drive/1vkp-uPV8tKavmX12bcN2L-jYH8_MgmHL?usp=sharing"><img src="https://img.shields.io/badge/-Colab%20Demo-blue" /></a>
-<a href="https://ncsu.qualtrics.com/jfe/form/SV_enuiA15WX8w74qy"><img src="https://img.shields.io/badge/-Caiman%20Survey-green" /></a>
+# CaImAn 
+<img src="docs/LOGOS/Caiman_logo_FI.png" width="400" align="right">
 
-CaImAn
-======
-<img src="https://github.com/flatironinstitute/CaImAn/blob/master/docs/LOGOS/Caiman_logo_FI.png" width="500" align="right">
+A Python toolbox for large-scale **Ca**lcium **Im**aging **An**alysis.    
 
-A Python toolbox for large scale **Ca**lcium **Im**aging data **An**alysis and behavioral analysis.
+CaImAn implements a set of essential methods required to analyze calcium and voltage imaging data. It provides fast and scalable algorithms for motion correction, source extraction, spike deconvolution, and registering neurons across multiple days. It is suitable for both two-photon and one-photon fluorescence microscopy data, and can be run in both offline and online modes. Documentation is [here](https://caiman.readthedocs.io/en/latest/). 
 
-CaImAn implements a set of essential methods required in the analysis pipeline of large scale calcium imaging data. Fast and scalable algorithms are implemented for motion correction, source extraction, spike deconvolution, and component registration across multiple days. It is suitable for both two-photon and one-photon fluorescence microscopy data, and can be run in both batch and online modes. CaImAn also contains some routines for the analysis of behavior from video cameras. A list of features as well as relevant references can be found [here](https://caiman.readthedocs.io/en/latest/CaImAn_features_and_references.html).
+# Quick start :rocket:
+To get started quickly, from installation to working through a demo notebook, take the following three steps:
 
-## Requirements
+### Step 1: Install caiman
+The following is all done in your anaconda prompt, starting in your base environment:
```

Comment:
This rewrite removed the first mentions of miniconda, installing it, and getting familiar with it.
A fresh user might be puzzled by this if they haven't heard of conda yet.

Reply:
I added a little bit about it, but most is now in the installation page